kairon/shared/pos/processor.py

```
@@ -409,6 +409,21 @@ def get_pos_products(self, session_id: str):
+    def invalidate_session(self, session_id):
+        url = f"{self.__base_url}/web/session/destroy"
+        cookies = {"session_id": session_id}
+        payload = {
+            "jsonrpc": "2.0",
+            "method": "call",
+            "params": {}
+        }
+        try:
+            response = requests.post(url, json=payload, cookies=cookies, timeout=30).json()
+            return response
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)